tests/cases/hash_state_test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2016 Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from .. import base
+from girder.utility import hash_state
+import hashlib
+import this
```

Comment:
Since ``import this`` has side effects (printing to stdout), let's move it inside the new ``setUp`` method. Scope shouldn't be an issue, since it's only used to create ``test_chunk`` (which should be renamed to ``this.testChunk``).